Cargo.toml

```diff
@@ -12,6 +12,10 @@ license = "MIT"
 keywords = ["monochrome", "unicode", "bitmap", "font"]
 categories = ["encoding"]
 exclude = [".gitignore"]
+build-std = ["core", "alloc"]
 
 [dev-dependencies]
 clap = { version = "4.5.3", features = ["derive"] }
+
+[dependencies]
+unicode-bidi = { version = "0.3.15", default-features = false, features = ["hardcoded-data"] }
```

Comment:
@Konosh93 The default features for `unicode-bidi` here include `std`, so to turn that off, we do `default-features = false`, then name the features we actually need explicitly.